apps/emqx/rebar.config.script

```diff
@@ -13,8 +12,13 @@ IsWin32 = fun() ->
                 win32 =:= element(1, os:type())
           end,
 
+IsMacOS = fun() ->
+               {unix, darwin} =:= os:type()
+          end,
+
 IsQuicSupp = fun() ->
-                not (IsCentos6() orelse IsWin32() orelse
+                not (IsCentos6() orelse IsWin32()
+                     orelse IsMacOS() orelse
```

Comment:
This'll have to be replicated here as well: https://github.com/emqx/emqx/blob/680ee9758a7d6e9293001bc816f1b1e43be3e3c6/mix.exs#L594-L600